lib/history.bash

```diff
@@ -1,12 +0,0 @@
-#!/usr/bin/env bash
-
-# Bash History Handling
-
-shopt -s histappend                                      # append to bash_history if Terminal.app quits
-export HISTCONTROL=${HISTCONTROL:-ignorespace:erasedups} # erase duplicates; alternative option: export HISTCONTROL=ignoredups
-export HISTSIZE=${HISTSIZE:-5000}                        # resize history size
-export AUTOFEATURE=${AUTOFEATURE:-true autotest}         # Cucumber / Autotest integration
-
-function rh {
```

Comment:
I think this needs to return, like we discussed. Something like `fav_commands`?

Reply:
haha don't worry, that's why i brought it up in the chat first :smile:

Reply:
I think we settled on a name now- if you change it, I will approve :smile:

Reply:
@nwinkler @NoahGorny have a look